garrysmod/lua/includes/extensions/vector.lua

```diff
@@ -3,10 +3,262 @@ local meta = FindMetaTable( "Vector" )
 -- Nothing in here, still leaving this file here just in case
 
 --[[---------------------------------------------------------
-	Converts Vector To Color - alpha precision lost, must reset
+Converts Vector to Color - alpha precision lost, must reset
 -----------------------------------------------------------]]
-function meta:ToColor( )
+function meta:ToColor(num)
+	local x, y, z = self:Unpack()
+	return Color(x * 255, y * 255, z * 255, (num or 0) * 255)

```

Comment:
This should be ``num == nil and 1 or num`` (the ``== nil`` to prevent people accidentally sending in ``false`` as the argument and never noticing the difference, but more importantly having 1 as the default so the alpha defaults to 255).

Reply:
Yes, thank you. Fixed ;)